bin/historical/migrations/_common.py

```diff
@@ -0,0 +1,41 @@
+import os
+import subprocess
+import importlib
+import logging
+
+import emission.core.get_database as edb
+
+DB_HOST_TEMPLATE = "mongodb://localhost:27017/openpath_prod_REPLACEME"
```

Comment:
```suggestion
DB_HOST_TEMPLATE = os.environ["DB_HOST_TEMPLATE"]
```

The template for DB_HOST on DocumentDB is very different. It has a different host (not `localhost`), and includes user+password

Reply:
Fixed in https://github.com/e-mission/e-mission-server/pull/1009/commits/947492d1b7ec509e7afc0f7b6af0a9eca52ac93a